marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-match-merge-writer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "role-name": "hub-central-match-merge-writer",
+  "description": "Permits reading/reviewing matching and merging steps in Hub Central",
+  "role": [
+    "hub-central-match-merge-reader",
+    "data-hub-common-writer",
+    "data-hub-match-merge-writer",
+    "data-hub-step-definition-reader"
```

Comment:
Also "data-hub-mapping-writer" and "data-hub-ingestion-writer" have "data-hub-step-definition-reader" . So, should this ("data-hub-step-definition-reader") be moved to "data-hub-match-merge-writer" ?

Reply:
Good catch. I think it would make more sense on the "data-hub-match-merge-writer". The "data-hub-step-definition-reader" is needed to read the step definition at the time it is saved.

Reply:
Added "data-hub-step-definition-reader" to  "data-hub-match-merge-writer"